src/backend/package.json

```diff
@@ -24,7 +25,8 @@
     "jsonwebtoken": "^8.5.1",
     "multer": "^1.4.5-lts.1",
     "nodemailer": "^6.9.1",
-    "shared": "1.0.0"
+    "shared": "1.0.0",
+    "vite": "^2.6.4"
```

Comment:
remove vite, can u remove express-pino-logger as well, anthony said it's unused now